CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing
+
+This document <https://github.com/gitops-working-group/gitops-working-group/blob/main/CONTRIBUTING.md> defines the contribution process for the GitOps Working Group project and community.
+These guidelines apply to all git repositories in the `gitops-working-group` GitHub org, however each repository may specify additional contribution processes.
+
+For more information on becoming involved in the Working Group in general, see [Getting Involved](https://github.com/gitops-working-group/gitops-working-group/blob/main/README.md#how-to-get-involved).
+
+### Pull Request Tips - Submitting and Reviewing
+
+- Talk to us via GitHub issue, mailing list, or meeting before you start on major
```

Comment:
```suggestion
Talk to us via GitHub [issue](https://docs.github.com/en/github/managing-your-work-on-github/about-issues), [discussion](https://docs.github.com/en/discussions), mailing list, or meeting before you start on major
```

I suggest to add _discussions_ to the options before starting a major effort. In case interested people don't know about _issues_ and _discussions_ I suggest to add documentation links:

- https://docs.github.com/en/github/managing-your-work-on-github/about-issues
- https://docs.github.com/en/discussions